errors/missingTrieNodeError.go

```diff
@@ -3,28 +3,9 @@ package errors
 import (
 	"encoding/hex"
 	"fmt"
-	"strings"
-
 	"github.com/multiversx/mx-chain-go/common"
 )
 
-// IsGetNodeFromDBError returns true if the provided error is of type getNodeFromDB
```

Comment:
maybe continue the cleanup on mx-chain-go? Remove the `GetNodeFromDBErrorString` and `IsClosingError` definitions and use the ones on the core?

Reply:
Missed them. Removed.